frescobaldi_app/main.py

```diff
@@ -189,6 +192,9 @@ def main():
         sys.stdout.write(debuginfo.version_info_string() + '\n')
         sys.exit(0)
 
+    if args.python_ly:
+        sys.path = [args.python_ly] + sys.path
```

Comment:
Another option is to write `sys.path.insert(0, args.python_ly)`

Reply:
I'll have to investigate this anyway. I noticed that when invoking Frescobaldi with that `-y` option it obviously found the `ly` module - but the `Git` menu was missing! And I have no idea why yet ...

Reply:
After some investigation it does seem to be a direct consequence of this code!

❝the first item of this list, path[0], is the directory containing the script that was used to invoke the Python interpreter.❞

This is also what it is used to check if git is present which in turn is a prerequisite for the git menu:
https://github.com/wbsoft/frescobaldi/blob/master/frescobaldi_app/vcs/__init__.py#L73

❝Notice that the script directory is inserted before the entries inserted as a result of PYTHONPATH.❞
https://docs.python.org/3/library/sys.html

So it seems you should be doing:
`sys.path.insert(1, args.python_ly)`
Please confirm this!

Reply:
Yes, that's correct. Thank you for investigating.
95b19711